packages/testlab/README.md

```diff
@@ -2,10 +2,19 @@
 
 A collection of test utilities we use to write LoopBack tests.
 
-## `expect`
+## Overview
 
-[Should.js](https://shouldjs.github.io/) configured in "as-function" mode
-(global `Object.prototype` is left intact) with an extra chaining word `to`.
+Test utilities to help writing loopback-next tests:
+- Should.js: expect(actual, expected, [message]).
+- sinon: Standalone test spies, stubs and mocks, that Works with any unit testing framework.
```

Comment:
I think we need some explanation of what "test spies" are... Stubs and mocks are perhaps better understood, but a short description would also help.  Also watch captialization (e.g. "...Works with...").

Oh, I see there's a link to http://sinonjs.org/ .  That may be sufficient, but I still think some more general info would help.